emacs/lsp-cquery.el

```diff
@@ -307,10 +313,5 @@
 (advice-add 'lsp--text-document-code-action-callback :around
             '(lambda (orig actions) (setq lsp-code-actions actions)))
```

Comment:
@topisani haven't had a chance to look into this one yet, but if you could elaborate briefly on what weird behaviour you were seeing / open an issue against lsp-mode, I can take a look.

Reply:
every time lsp-mode recieves codeactions from cquery, it just adds them to an ever growing list of codeactions, instead of replacing the ones that were already there. But since they dont have a complete implementation of codeactions yet, that might be a mistake

Reply:
That… does kinda seem like a mistake. Guess we'll have to watch what happens with the upstream development there.